tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,40 @@
+Feature: preview 3D model
+    As a user
+    I want to preview a 3D model
+    So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+
+    Scenario: preview 3D model
+        Given the user has logged in with username "admin" and password "admin"
+        And the user has uploaded the following 3D models:
+            | model1.glb |
+            | model2.glb |
+
+        # preview 3D model
+        When the user opens the file "model1.glb" in the 3D model viewer
+        Then the 3D model will be display in the viewport
+        And the file name "model1.glb" will be shown in the topbar
+
+        # toggle between standard and fullscreen mode
+        When the user clicks on “Enter full screen mode”
```

Comment:
There are weird quotes, do they actually work?
```suggestion
        When the user clicks on "Enter full screen mode"
```

Reply:
They are the the quotation marks that are usually used in written English (UK), see https://en.wikipedia.org/wiki/Quotation_mark#Summary_table & https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html

I don't know if the code would pass, haven't implemented the test code yet, but it's good to know that for consistency we use the vertical quotation mark (U+0022) and vertical apostrophe sign (U+0027)